src/redivis/common/list_rows.py

```diff
@@ -70,7 +70,8 @@ def list_rows(
     target_parallelization=None,
     progress=True,
     coerce_schema = False,
-    batch_preprocessor = None
+    batch_preprocessor = None,
+    folder = None,
```

Comment:
list_rows doesn't currently accept a `folder` param (I imagine you added this while debugging), so we should remove this.